_security-plugin/configuration/ldap.md

```diff
@@ -473,8 +475,7 @@ skip_users:
   - '/\S*/'
 ```
 
-
-### (Advanced) Exclude roles from nested role lookups
+### Exclude roles from nested role lookups
 
 If the users in your LDAP installation have a large number of roles, and you have the requirement to resolve nested roles as well, you might run into performance issues.
```

Comment:
Suggested rewrite to tighten wording: If the users in your LDAP installation have several roles and you have requirements to resolve nested roles, you might encounter performance issues.

Reply:
@vagimeli Yes, good suggestion. I can incorporate that here. There is, however, another PR for general LDAP updates, and I'll focus on polishing language there. This PR is mainly for pooling settings.